main.js

```diff
@@ -847,50 +846,26 @@ function updatePersistentDataForm(iniobj) {
 // Update an ini object from the persistent data form.
 function updateIniFromForm(ini) {
     "use strict";
-    ini.General.Name = document.getElementById("ini-name").value;
-    ini.General.Room = document.getElementById("ini-location").value;
-    ini.General.Kills = document.getElementById("ini-kills").value;
-    ini.General.Love = document.getElementById("ini-love").value;
-    if (document.getElementById("ini-omega-flowey-trapped").checked) {
-        if (!ini.FFFFF) {
-            ini.FFFFF = {};
-        }
-        ini.FFFFF.F = "1";
-    } else {
-        if (ini.FFFFF) {
-            ini.FFFFF.F = "0";
-        }
-    }
-    var upcomingSoul = Number(document.getElementById("ini-omega-flowey-soul").value);
-    if (upcomingSoul) {
-        if (!ini.FFFFF) {
-            ini.FFFFF = {};
-        }
-        ini.FFFFF.P = upcomingSoul;
-    }
-    var timesDied = Number(document.getElementById("ini-omega-flowey-deaths").value);
-    if (timesDied) {
-        if (!ini.FFFFF) {
-            ini.FFFFF = {};
-        }
-        if (ini.FFFFF) {
-            ini.FFFFF.D = timesDied;
-        }
-    }
-    if (document.getElementById("ini-dodged-all-special-thanks").checked) {
-        if (!ini.reset) {
-            ini.reset = {};
+    
+    for (var i in iniIDs) { // 0 -> element ID, 1 -> ini section, 2 -> ini key
```

Comment:
Use `forEach` or `for ... of` to iterate over an array:
```javascript
for (const [elementID, iniSection, iniKey] of iniIDs) {
    // ...
}
```
(now that we're IE-incompatible, anyways)